core/templates/components/shared-component.module.ts

```diff
@@ -118,7 +120,8 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     ExplorationEmbedButtonModalComponent,
     KeyboardShortcutHelpModalComponent,
     SkillMasteryViewerComponent,
-    SocialButtonsComponent
+    SocialButtonsComponent,
+    SummaryListHeaderComponent
```

Comment:
Add`SummaryListHeaderComponent` to the exports array.

Reply:
Done